### ISSUE TITLE
AVT 3 - React FileUploader Component, VO fails to read X icon on macOS

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3438]
## Environment
macOS 10.14.5 with VoiceOver
Chrome Version 75.0.3770.100 (Official Build) (64-bit)

## Description
1. Tab to the Add Files button, VO announces, "Add Files  button"
2. Press Enter or Space to open the dialog, navigate to a jpg file and select it.
3. Press Tab to the X icon button, VO announces "Provide icon description".
**Expected Result:** VO should announce the purpose of the button, such as "Clear File"
Note: Pressing the button clears the file, but the user is unaware that is happening.
